FirebaseCore/Sources/Public/FirebaseCore/FIROptions.h

```diff
@@ -32,7 +32,7 @@ NS_SWIFT_NAME(FirebaseOptions)
 
 /**
  * An iOS API key used for authenticating requests from your app, e.g.
- * @"AIzaSyDdVgKwhZl0sTTTLZ7iTmt1r3N2cJLnaDk", used to identify your app to Google servers.
+ * The key contains about 39 alphanumeric digits, used to identify your app to Google servers.
```

Comment:
There may be more but I believe the following are true: "The key must begin with "A" and contain exactly 39 alphanumeric characters".

cc: @ryanwilson